kroxylicious-test-tools/src/main/java/io/kroxylicious/test/tester/KroxyliciousTester.java

```diff
@@ -209,6 +213,73 @@ public interface KroxyliciousTester extends Closeable {
      */
     KafkaClient simpleTestClient(String virtualCluster);
 
+    /**
+     * Creates the given topics on the specified virtual cluster, with an admin client using the specified configuration.
+     * @param topics the topics to be created
+     * @param additionalConfig additional admin config
+     * @param virtualCluster the virtual cluster we want the topics created on
+     * @return CreateTopicsResult
+     */
+    CreateTopicsResult createTopics(String virtualCluster, Map<String, Object> additionalConfig, NewTopic... topics);
+
+    /**
+     * Creates the given topic on the specified virtual cluster, with an admin client using the specified configuration.
+     * @param topic the name of the topic to be created
+     * @param numPartitions the number of partitions for the topic
+     * @param additionalConfig additional admin config
+     * @param virtualCluster the virtual cluster we want the topics created on
+     * @return CreateTopicsResult
+     */
+    CreateTopicsResult createTopic(String virtualCluster, Map<String, Object> additionalConfig, String topic, int numPartitions);
```

Comment:
Wondering if 4 overloads is enough to justify looking at a builder to cut down the ways we can create.

```
tester.createTopic(CreateTopic.named("xyz"));
tester.createTopic(CreateTopic.named("xyz").withPartitions(3));
tester.createTopic(CreateTopic.from(new NewTopic(x, y, z)));
tester.createTopic(CreateTopic.named("xyz").withConfig(Map.of()));
```

Reply:
Maybe, though there's 6 overloads each for producer, and consumer methods, and 4 overloads for admin methods on the tester already. I've got no problem implementing a builder for topics if it makes things easier for people using it, just wanted to point that out.

Reply:
Actually now that I think about it, the reason for most of these overloaded methods is because of convenience with `Admin` creation, not anything to do with topics themselves. Both `String virtualCluster` and `Map<String, Object> additionalConfig` arguments are for `Admin` creation and are passed straight into that. The only convenience overloads we're doing for topics specifically are `NewTopic... topics` vs `String topic, int numPartitions`, which is really just so the test doesn't have to construct it's own `NewTopic` if it doesn't want to.